### ISSUE TITLE
floatingLabel doesn't float when using `React.forwardRef`

### ISSUE THREAD

**Issue by theapache64** [GeekyAnts/NativeBase#2037]
## react-native, react and native-base version
```
"dependencies": {
    "react": "16.3.1",
    "react-native": "0.55.4",
    "native-base": "^2.5.2"
  },
```
## Expected behaviour
`floatingLabel` should float
## Actual behaviour
`floatingLabel` doesn't float
## Steps to reproduce (code snippet or screenshot)

Clone [this](https://github.com/theapache64/NativeBaseForwardRefTest) project

## Screenshot of emulator/device
![image](https://user-images.githubusercontent.com/9678279/41654519-55fd718a-74a7-11e8-89fe-3b0e1202a6ed.png)

## Is the bug present in both ios and android or in any one of them?
Yes, both

## Any other additional info which would help us debug the issue quicker.
Is it because the `react` version that `NativeBase` uses ?

**Comment by akhil-geekyants** [comment]
@theapache64 checked this. Floating label is working fine with `React.forwardRef`. Attaching a gif.
 
Code
```
import React, { Component } from 'react';
import { Container, Header, Content, Form, Item, Input, Label, Text, Button } from 'native-base';

export default class App extends Component {

  render() {
    return (
      <Container>
        <Header />
        <Content>
          <Form>
            <MyFloatingLabel ref={c => this.username = c} label={"Username"} />
            <MyFloatingLabel ref={c => this.password = c} label={"Password"} />
          </Form>
          <Button style={{ alignSelf: "center", marginTop: 20 }} onPress={() => this.password._root.focus()}><Text>Change focus</Text></Button>
        </Content>
      </Container >
    );
  }
}

const MyFloatingLabel = React.forwardRef((props, ref) => (
  <Item floatingLabel>
    <Label>{props.label}</Label>
    <Input getRef={ref} />
  </Item>
));

```
Gif

![forwardref](https://user-images.githubusercontent.com/27768184/41661419-5aa71a28-74bc-11e8-9c9f-329d1cb94573.gif)

**Comment by theapache64** [comment]
I've copy pasted the code, and I am getting the same error. 

![image](https://user-images.githubusercontent.com/9678279/41662651-31dbd176-74bf-11e8-9992-6f6ce14bcc84.png)

Here's my full [package.json](https://raw.githubusercontent.com/theapache64/NativeBaseForwardRefTest/master/package.json)

**Comment by akhil-geekyants** [comment]
Related issue #2000

**Comment by theapache64** [comment]
In #2000 `floatingLabel` takes time to float. but in this case, it's doesn't float.

**Comment by theapache64** [comment]
[https://snack.expo.io/BkwuQ1u-7](Reproduced snack link)

**Comment by theapache64** [comment]
I've also found that the changing of version `native-base` version to `v2.4.5` fixed the issue. [See here](https://snack.expo.io/HyQbOgOZQ)

**Comment by akhil-geekyants** [comment]
@theapache64 expecting PR https://github.com/GeekyAnts/NativeBase/pull/2030 to solve the issue

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.6.1](https://github.com/GeekyAnts/NativeBase/blob/v2.6.1/CHANGELOG.md)

**Comment by theapache64** [comment]
Perfect. @akhil-geekyants good work (y)
